airflow/dags/ingestion/ask-astro-load-cosmos-docs.py

```diff
@@ -0,0 +1,59 @@
+import os
+from datetime import datetime
+
+from include.utils.slack import send_failure_notification
+
+from airflow.decorators import dag, task
+from airflow.providers.weaviate.operators.weaviate import WeaviateDocumentIngestOperator
+
+ask_astro_env = os.environ.get("ASK_ASTRO_ENV", "dev")
+
+_WEAVIATE_CONN_ID = f"weaviate_{ask_astro_env}"
+WEAVIATE_CLASS = os.environ.get("WEAVIATE_CLASS", "DocsDev")
+
+
+default_args = {"retries": 3, "retry_delay": 30}
+
+schedule_interval = os.environ.get("INGESTION_SCHEDULE", "0 5 * * 2") if ask_astro_env == "prod" else None
+
+
+@task
+def get_cosmos_docs_content():
+    from include.tasks.extract.cosmos_docs import extract_cosmos_docs
+
+    return extract_cosmos_docs()
```

Comment:
Instead of adding a wrapper around what effective is a TaskFlow function, WDYT about decorating the `extract_cosmos_docs` function with the `@task` decorator straight away?